playbooks/templates/openstack_user_config.yml.j2

```diff
@@ -0,0 +1,199 @@
+---
+cidr_networks:
+  container: 10.0.236.0/22
+  tunnel: 10.0.240.0/22
+  storage: 10.0.244.0/22
+  flat: 10.0.248.0/22
```

Comment:
maybe we should make the base cidrs configurable upstream?

Reply:
with pr https://review.openstack.org/#/c/507886/ i think this file could be dropped.

Reply:
the previous pr has merged.

Reply:
this pr has merged